pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "funding-service-design-utils"
-version = "2.0.6"
```

Comment:
Why does this say `2.0.6` - main says `2.0.7`...? 

https://github.com/communitiesuk/funding-service-design-utils/blob/main/pyproject.toml#L7

Outdated branch?

Reply:
Fixed in https://github.com/communitiesuk/funding-service-design-utils/pull/87/commits/27afb6545558fe4389ce4e9eebe0f3fe786e3408